__tests__/screens/FeedScreen/HeaderFeed.test.js

```diff
@@ -65,7 +65,7 @@ describe('Header feed should run correctly', () => {
     expect(getAllByTestId('haveBackButton')).toHaveLength(1);
     expect(allIdAnonym('haveBackButton')).toHaveLength(1);
     fireEvent.press(idAnonym('onBack'));
-    expect(mockedNavigate).toHaveBeenCalledTimes(1);
+    expect(mockedGoBack).toHaveBeenCalledTimes(1);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `mockedNavigate` to `mockedGoBack` seems appropriate as it aligns with the back button functionality being tested. The test now correctly asserts that the `goBack` function is called when the back button is pressed.
```diff
-    expect(mockedNavigate).toHaveBeenCalledTimes(1);
+    expect(mockedGoBack).toHaveBeenCalledTimes(1);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->